README.adoc

```diff
@@ -10,7 +10,7 @@ All OpenShift documentation is sourced in http://www.methods.co.nz/asciidoc/[Asc
 The documentation published from these source files can be viewed at http://docs.openshift.com.
 
 == Contributing to OpenShift documentation
-If you are interested in contributing to OpenShift technical documentation, you can view all our link:../contributing_to_docs[resources] that will help you get set up and provide more information. 
+If you are interested in contributing to OpenShift technical documentation, you can view all our link:./contributing_to_docs[resources] that will help you get set up and provide more information. 
 
```

Comment:
A leading `./` or `/` both work (the latter style appears to be what was used elsewhere in these topics), as does removing any leading characters entirely. I don't think it really matters though (so long as the links aren't broken) since these files aren't built in the docs themselves and the links are mainly just for navigating around on GH. But :+1: for fixing the broken link!